src/distribute.js

```diff
@@ -0,0 +1,186 @@
+        CONTRACT_ADDRESS,
+        {
+            defaultGasPrice: '500000000000'
+        }
+    )
+};
+
+const isProject = (csvCategory) => {
+    return !(isPrivateSale(csvCategory) || isPublicSale(csvCategory))
+};
+
+const isPrivateSale = (csvCategory) => {
+    return csvCategory === CATEGORY_CSV_SALE_PRIVATE;
+};
+
+const isPublicSale = (csvCategory) => {
+    return csvCategory === CATEGORY_CSV_SALE_PUBLIC;
+};
+
+const calculateDepositValue = (purchasedBn, bonusBn) => {
+    return purchasedBn.add(bonusBn);
+};
+
+const logDeposit = (wallet, to, value, category) => {
+    console.log("Depositing:");
+    console.log(`   from: ${wallet}`);
+    console.log(`   to: ${to}`);
+    console.log(`   value: ${value}`);
+    console.log(`   category: ${category}`);
+};
+
+const csvCategoryToSolidityEnumValue = (csvCategory) => {
+    switch (csvCategory) {
+        case CATEGORY_CSV_PROJECT_TEAM:
+            return CATEGORY_SOLIDITY_ENUM_TEAM;
+        case CATEGORY_CSV_PROJECT_SEED_CONTRIBUTORS:
+            return CATEGORY_SOLIDITY_ENUM_SEED_CONTRIBUTORS;
+        case CATEGORY_CSV_PROJECT_CONSULTANTS:
+            return CATEGORY_SOLIDITY_ENUM_CONSULTANTS;
+        case CATEGORY_CSV_PROJECT_OTHER:
+            return CATEGORY_SOLIDITY_ENUM_OTHER;
+        case CATEGORY_CSV_PROJECT_FUTURE_OFFERING:
+            return CATEGORY_SOLIDITY_ENUM_FUTURE_OFFERING;
+        case CATEGORY_CSV_SALE_PRIVATE:
+            throw 'CSV_CATEGORY_PRIVATE_SALE_HAS_NO_SOLIDITY_ENUM_VALUE';
+        case CATEGORY_CSV_SALE_PUBLIC:
+            throw 'CSV_CATEGORY_PUBLIC_SALE_HAS_NO_SOLIDITY_ENUM_VALUE';
+        default:
+            throw 'CSV_CATEGORY_UNKNOWN';
+    }
+};
+
+const transfer1PhtToEveryDistributionAddress = async (data) => {
+    data.forEach(async function (distribution) {
+        console.log(`Transferring 1 PHT to ${distribution.to}...`);
+
+        await web3.eth.sendTransaction({
+            from: WALLET_PROJECT,
+            to: distribution.to,
+            value: pht2Wei('1'),
+            gas: 21000
+        });
+    });
+
+    console.log("All distribution accounts were topped-up!");
+};
+
+csv()
+.fromFile(CSV_FILEPATH)
+.then(async (data) => {
+    const txReceipts = [];
+    const contract = initContract();
+
+    try {
+        await transfer1PhtToEveryDistributionAddress(data);
+    } catch (e) {
+        process.exit(1);
+    }
+
+    data.forEach(async function (distribution) {
+        const gasLimit = '300000';
+
+        const to = distribution.to;
+        const purchased = pht2Wei(distribution.purchased_pht);
+        const bonus = pht2Wei(distribution.bonus_pht);
+        const category = distribution.category;
+        const value = calculateDepositValue(purchased, bonus);
+
+        if (isProject(category)) {
+            logDeposit(WALLET_PROJECT, to, value, category);
+
+            const solidityCategory = csvCategoryToSolidityEnumValue(category);
+
+            contract.methods
```

Comment:
[MEDIUM] You are missing Tx send `.catch ()` which occur when tx is not being sent due to js checks.

See an example in the [official doc](https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#id26)